lib/pivotal_tracker.rb

```diff
@@ -7,9 +7,11 @@ class PivotalTracker
 require 'pivotal-tracker/client'
 require 'pivotal-tracker/story'
 require 'pivotal-tracker/project'
+require 'pivotal-tracker/account'
```

Comment:
Should we use autoload here?  http://www.rubyinside.com/ruby-techniques-revealed-autoload-1652.html